sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collectionOperations.scala

```diff
@@ -44,6 +46,9 @@ case class Size(child: Expression) extends UnaryExpression with ExpectsInputType
  * Sorts the input array in ascending / descending order according to the natural ordering of
  * the array elements and returns it.
  */
+@ExpressionDescription(
+usage = "_FUNC_(array, order) - Sorts the input array for the given column in " +
+  "ascending / descending order, according to the natural ordering of the array elements")
```

Comment:
Maybe it is better to use `_FUNC_(array, ascendingOrder)` and mention that `ascendingOrder` is a boolean?